src/main/java/com/swp/common/domain/users.java

```diff
@@ -0,0 +1,27 @@
+package com.swp.common.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+public class users {
+    @Id
+    @Column(name = "user_id",nullable = false)
+    private Long user_id;
```

Comment:
user_id가 mysql에서 auto_increment primary key 이니까 
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "user_id",nullable = false)
    private Long user_id;
```